src/test/java/edu/cornell/kfs/module/cam/document/service/impl/CuAssetSubAccountServiceImplTest.java

```diff
@@ -1,241 +1,224 @@
+        } else {
+            assertNull("Plant Detail should have had its sub-account cleared out", plantDetail.getSubAccountNumber());
+        }
+    }
+
+
+    protected <T> T createMockServiceExpectingNoCalls(Class<T> serviceClass) {
+        T mockService = EasyMock.createMock(serviceClass);
+        EasyMock.replay(mockService);
+        return mockService;
+    }
+
+    protected void setupParameterServiceWithPlantAccountValue(String parameterValue, EvaluationOperator operator) {
+        Parameter parameter = createAssetTransferPlantAccountParameter(parameterValue, operator);
+        assetSubAccountService.setParameterService(createMockParameterServiceForReturningFullParameter(parameter, parameter, Asset.class));
+    }
+
+    protected void setupParameterServiceToReturnNullPlantAccountParameter() {
+        Parameter parameterForSearchArgs = createAssetTransferPlantAccountParameter(KFSConstants.EMPTY_STRING, EvaluationOperator.DISALLOW);
+        assetSubAccountService.setParameterService(createMockParameterServiceForReturningFullParameter(parameterForSearchArgs, null, Asset.class));
+    }
+
+    protected ParameterService createMockParameterServiceForReturningFullParameter(Parameter searchArgs, Parameter returnValue, Class<?> componentClass) {
+        ParameterService parameterService = EasyMock.createMock(ParameterServiceImpl.class);
+        EasyMock.expect(parameterService.getParameter(componentClass, searchArgs.getName())).andStubReturn(returnValue);
+        EasyMock.expect(parameterService.getParameter(
+                searchArgs.getNamespaceCode(), searchArgs.getComponentCode(), searchArgs.getName())).andStubReturn(returnValue);
+        EasyMock.replay(parameterService);
+        return parameterService;
+    }
+
+    protected Parameter createAssetTransferPlantAccountParameter(String value, EvaluationOperator operator) {
+        Parameter.Builder parameter = Parameter.Builder.create(
+                KFSConstants.APPLICATION_NAMESPACE_CODE, CamsConstants.CAM_MODULE_CODE, Asset.class.getSimpleName(),
+                CuCamsConstants.Parameters.ASSET_PLANT_ACCOUNTS_TO_FORCE_CLEARING_OF_GLPE_SUB_ACCOUNTS,
+                ParameterType.Builder.create(KfsParameterConstants.PARAMETER_CONFIG_TYPE_CODE));
+        parameter.setValue(value);
+        parameter.setEvaluationOperator(operator);
+        return parameter.build();
+    }
+
+    protected AssetGlpeSourceDetail createSourceDetail(String accountNumber, String subAccountNumber) {
+        // This only sets up a few minimal fields. If testing other functionality of the service, more initialization is needed.
+        AssetGlpeSourceDetail sourceDetail = new AssetGlpeSourceDetail();
+        sourceDetail.setChartOfAccountsCode("IT");
+        sourceDetail.setAccountNumber(accountNumber);
+        sourceDetail.setSubAccountNumber(subAccountNumber);
+        return sourceDetail;
+    }
+
+
```

Comment:
not a huge deal, but there are extra lines of whitespace here (and a few other places in these classes I noticed); I don't think more than one blank line is really necessary